server/middleware/authorization.js

```diff
@@ -0,0 +1,24 @@
+const jwt = require("jsonwebtoken");
+require("dotenv").config();
```

Comment:
This does work like this, but I've usually seen the "dotenv" config loaded higher up in the entry point to the app, so in `server.js` (or sometimes even in the webpack config). That means you have access to the `process.env.*` early on and don't have to worry about loading it again.

Reply:
done!